source/guides/maintenance/cpu-performance.rst

```diff
@@ -193,6 +193,53 @@ group:
    After adding a new group, you must log out and log back in for the new group
    to take effect.
 
+Thermal configuration
+=====================
+
+Better thermal control and performance can be archived by providing platform
+specified configuration to thermald.
+
+`Linux DPTF Extract Utility`_ is a companion tool to thermald, This tool can
+make use of "Intel ® Dynamic Platform and Thermal Framework (`Intel DPTF`_)"
+technology, and convert to the thermal_conf.xml configuration format used
+by thermald.
+
+First, make sure thermald version is equal or above 1.9.1:
+
+.. code:: bash
+
+   thermald --version
+
+Otherwise, you need to add an option :command:`--ignore-default-control` to
+thermald:
+
+.. code:: bash
+
+   # edit thermald.servcie
+   vim /usr/lib/systemd/system/thermald.service
+   # append --ignore-default-control option to ExecStart line, like below
+   # ExecStart=/usr/bin/thermald --no-daemon --dbus-enable --ignore-default-control
+
+   # reload
+   sudo systemctl daemon-reload
+
+Then, generate thermal configuration as below:
+
+.. code:: bash
+
+   git clone https://github.com/intel/dptfxtract.git
+   cd dptfxtract
+   sudo acpidump > acpi.out
+   acpixtract -a acpi.out
+   sudo ./dptfxtract *.dat
+
+thermald configuration files will be generated and saved to
+:command:`/etc/thermal/` folder. Restart thermald service to take effect.
+
+.. code:: bash
+
+   sudo systemctl restart thermald.service
+
```

Comment:
After running this command, I ran again the command ``thermald --version``. The output was ```1.9``, not ``1.9.1``. 

Should ``thermald`` have been upgraded? Or, was this prevented due to the error on "tables" from above?   
Can you include a command that will confirm that the above changes took effect? In other words: What change should be noticeable from the standpoint of a user?

Reply:
No, modifying `thermald.service` will not change `thermald` version, it's a workaround for version under `1.9`.

User can check `thermald` status by `systemctl status thermald.service`, and find log like:
```
Dec 06 05:44:54 icl-clr-debug-i5 thermald[459]: [WARN]Using generated /etc/thermald/thermal-conf.xml.auto
```
it indicates that the configure file has been used by `thermald`.

Reply:
That's fair.  Please include a statement to that effect. Something along the lines of "This is a workaround..."

Reply:
I just remove this workaround part, and suggest user to upgrade to latest Clear Linux version.

Reply:
OK. Please add the command to update, shown on Line 219 above.